.env.example

```diff
@@ -1,5 +1,8 @@
-# Release version e.g. 2022q2 or simply latest
-RELEASE=2024q1
+# Select flavor: care, research, dev
+FLAVOR=care
```

Comment:
I would suggest naming this differently. Maybe `mtb`?

Reply:
done.